charts/aws-ebs-csi-driver/values.yaml

```diff
@@ -227,6 +227,7 @@ controller:
     create: true
     name: ebs-csi-controller-sa
     annotations: {}
+    automountServiceAccountToken: false
```

Comment:
@kahirokunn you forgot to change this one to true too

Reply:
sorry, fixed.